internal/presenters/presenter_local_finding.go

```diff
@@ -139,34 +129,68 @@ func LoadTemplates(files []string, tmpl *template.Template) error {
 	return nil
 }
 
-func (p *LocalFindingPresenter) Render() (string, error) {
-	localFindingsTemplate, err := template.New("local_finding").Parse("")
+func mergeMaps[K comparable, V any](mapA map[K]V, mapB map[K]V) map[K]V {
```

Comment:
**nit:** Could we move this to a util method to improve discoverability to encourage reuse in future

Reply:
sure, I can move it